docs/topics/serialization.txt

```diff
@@ -328,9 +328,10 @@ YAML serialization looks quite similar to JSON. The object list is serialized
 as a sequence mappings with the keys "pk", "model" and "fields". Each field is
 again a mapping with the key being name of the field and the value the value::
 
-    -   fields: {expire_date: !!timestamp '2013-01-16 08:16:59.844560+00:00'}
-        model: sessions.session
+    -   model: sessions.session
         pk: 4b678b301dfd8a4e0dad910de3ae245b
+        fields:
+            expire_date: 2013-01-16 08:16:59.844560+00:00
```

Comment:
I'm getting the same output with `PyYAML` < 5.4. Also, it depends on YAML dumper, see [comment](https://code.djangoproject.com/ticket/12914#comment:14 ). I cannot reproduce your output even with the newest `PyYAML`  :thinking: 
```
  fields: {expire_date: ! '2013-01-16 08:16:59.844560+00:00'}
```

Reply:
did you try it using `serializers.serialize` function without any additional arguments, like `default_flow_style=None`?